chain/chain/src/chain.rs

```diff
@@ -383,20 +382,9 @@ impl Chain {
 
     pub fn compute_bp_hash_inner<'a, T>(bps: T) -> Result<CryptoHash, Error>
```

Comment:
It seems that you don't really need this function given the way it is right now

Reply:
It's still called from three different places (test, block production and genesis)